components/org.wso2.carbon.identity.conditional.auth.functions.common/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/common/utils/Constants.java

```diff
@@ -46,4 +47,40 @@ public class Constants {
     public static final String HTTP_FUNCTION_ALLOWED_DOMAINS = "AdaptiveAuth.HTTPFunctionAllowedDomains.Domain";
     public static final String CHOREO_DOMAINS = "AdaptiveAuth.ChoreoDomains.Domain";
     public static final String CHOREO_TOKEN_ENDPOINT = "AdaptiveAuth.ChoreoTokenEndpoint";
+
+    /**
+     * Define logging constants.
+     */
+    public static class LogConstants {
+
+        public static final String ADAPTIVE_AUTH_SERVICE = "adaptive-auth-service";
+        public static final String FAILED = "FAILED";
+
+        /**
+         * Define action IDs for diagnostic logs.
+         */
+        public static class ActionIDs {
+
+            public static final String RECEIVE_TOKEN = "receive-token";
+            public static final String RECEIVE_API_RESPONSE = "receive-api-response";
+        }
+
+        /**
+         * Define common and reusable Input keys for diagnostic logs.
+         */
+        public static class InputKeys {
+
+            public static final String TOKEN_ENDPOINT = "token endpoint";
+            public static final String EXTERNAL_API = "external api";
```

Comment:
shall we just use as "api" as the term "external api" is used within messages

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/778ee19b0b9677f6d2cf96e829414272e804de01